.circleci/test-scripts/baseline/compare-and-update-baseline.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+pip install --upgrade scipy google-cloud-storage
+
+jq -s 'flatten' "${WORKSPACE_ROOT}"/*perf.json > "${WORKSPACE_ROOT}"/all-perf.json
+
+export BASELINE="${CI_ROOT}/test-scripts/baseline/"
+"${BASELINE}"/main.py --test "${WORKSPACE_ROOT}"/all-perf.json \
+    | sort \
+    | awk -f "${BASELINE}"/format.awk > "${WORKSPACE_ROOT}"/benchmark.md
+
+"${BASELINE}"/main.py --update "${WORKSPACE_ROOT}"/all-perf.json
+
+# The baseline has to be updated only in the main branch, uncomment that before
+# merging.
+#
+#if [[ "$CIRCLE_BRANCH" == "master" || -n "$CIRCLE_TAG" ]]; then
+    #${BASELINE}/main.py --update ${WORKSPACE_ROOT}/all-perf.json
+#fi
```

Comment:
Why was this left as a comment? Is it replacing line 13 before merging the PR?

Reply:
Yep, that's the goal.